src/EditorFeatures/CSharpTest/OnTheFlyDocs/OnTheFlyDocsUtilitiesTests.cs

```diff
@@ -64,23 +51,14 @@ class A
             }
             """;
 
-        using var workspace = EditorTestWorkspace.CreateCSharp(testCode);
-        var solution = workspace.CurrentSolution;
-        var document = solution.Projects.First().Documents.First();
-
-        var syntaxTree = await document.GetSyntaxTreeAsync();
-        var semanticModel = await document.GetSemanticModelAsync();
-
-        var methodDeclaration = syntaxTree!.GetRoot()
-            .DescendantNodes()
-            .OfType<MethodDeclarationSyntax>()
-            .First();
-
-        var methodSymbol = semanticModel!.GetDeclaredSymbol(methodDeclaration);
+        var actualSymbolText = """
+            class A
+            {
+                public int x;
+            }
+            """;
 
-        var result = OnTheFlyDocsUtilities.GetAdditionalOnTheFlyDocsContext(solution, methodSymbol!);
-        Assert.NotNull(result.First());
-        Assert.Null(result.Last());
+        await TestMethodContextAsync(testCode, actualSymbolText);
```

Comment:
the test name is TestAdditionalContextWithTypeParameters, but i don't see type parameters here.